build.sbt

```diff
@@ -175,6 +175,7 @@ lazy val shared = (project in file("modules/shared"))
       Libraries.guava,
       Libraries.log4cats,
       Libraries.logback % Runtime,
+      Libraries.mapref,
```

Comment:
so I think you can remove it from core

Reply:
But core also uses this lib.

Reply:
@tbekas shouldn't it use it through this helpers?

Reply:
I don't think so. We import `import io.chrisdavenport.mapref.MapRef` in core in couple of places, therefore I think that this dependency should stay.